src/index.js

```diff
@@ -17,7 +17,10 @@ export type MongoOptionsType = {
     updatedAt: string,
   },
   convertUserIdToMongoObjectId: boolean,
-  caseSensitiveUserName: boolean
+  convertSessionIdToMongoObjectId: boolean,
+  caseSensitiveUserName: boolean,
+  idProvider: ?(() => string | Object),
+  dateProvider: () => any,
```

Comment:
dateProvider should probably accept optional date object like so:
`(date?: Date) => any`